### PR TITLE
Improve grammar of the doc

### PR DESCRIPTION
**Pull request by pmbaldha**

### CODE REVIEW

File: docs/gcp/google-app-engine.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Google App Engine
 
-You need to setup App Engine before you can deploy to GCP or use GCP resources for local development.
+You need to setup an App Engine before you can deploy to GCP or use GCP resources for local development.
```
Comment (jeffpaul) [inline docs/gcp/google-app-engine.md]: This change does not seem to make the text more legible or accurate, please revert this change.

**Review by jeffpaul — CHANGES_REQUESTED** [review]
@pmbaldha I've requested a small revert, otherwise these changes look good... thanks!

File: docs/gcp/google-app-engine.md (hunk 1 above)
Comment (pmbaldha) [reply docs/gcp/google-app-engine.md]: @jeffpaul fixed it. Thank you

**Comment by pmbaldha** [comment]
@jeffpaul Done.

**Review by jeffpaul — APPROVED** [review]
Looks good, thanks!
